LandSandBoat/server
# [chore] Utilize dynamic lookup for Mobs and NPCs with same naming convention, apply styling

**_I affirm:_**

## What does this pull request do?
* Converts a large chunk of NPCs and Mobs to use `DYNAMIC_LOOKUP`
* Applys consistent styling to all IDs.lua files



## Steps to test these changes
* Run DB query for each name in IDs.lua that was converted
* See no startup errors when looking up those IDs
